tests/e2e/tests/devfiles/JavaMaven.spec.ts

```diff
@@ -22,20 +22,20 @@ const codeNavigationClassName: string = 'String.class';
 const stack : string = 'Java Maven';
 const taskName: string = 'maven build';
 
-suite('Java Maven test', async () => {
+suite(`${stack} test`, async () => {
     suite (`Create ${stack} workspace ${workspaceName}`, async () => {
         workspaceHandling.createAndOpenWorkspace(workspaceName, stack);
         projectAndFileTests.waitWorkspaceReadiness(workspaceName, sampleName, 'src');
     });
 
     suite('Validation of workspace build and run', async () => {
-        codeExecutionTests.runTask(taskName, 120000);
```

Comment:
+1 Nice possibility, timeout is more readable.

Reply:
I'm not sure I understand what would you want me to do here. Could you please explain?

Reply:
I said that you made a good improvement, doesn't require any actions.  )

Reply:
oh, thank you :) I thought you wanted me to change something here :D